packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -586,9 +629,9 @@ Elsewhere in your file, using the method above, you can inspect the client ID an
 
 The Okta-hosted sign-in page uses the [Sign-In Widget](https://github.com/okta/okta-signin-widget#okta-hosted-sign-in-page-default) component to interact with the user.
 
-If you aren't familiar with the Sign-In Widget, Okta recommends that you select the highest **Major Version** and the latest **Minor Version** (default). For details about the Sign-In Widget capabilities that are supported by major and minor versions, see the [GitHub releases page](https://github.com/okta/okta-signin-widget/releases).
+If you aren't familiar with the Sign-In Widget, Okta recommends that you select the highest **Major Version** and the latest **Minor Version** (default). For details major and minor versions and the Sign-In Widget capabilities they support, see the [GitHub releases page](https://github.com/okta/okta-signin-widget/releases).
```

Comment:
Should this be:

"For details, major and minor versions, and the Sign-In Widget capabilities they support..."? 

The first bit seems like it's missing punctuation as "For details major and minor versions and the SIW capabilities" seems like it needs to be broken up a bit.

Reply:
done